src/vendorcode/intel/Makefile.inc

```diff
@@ -14,6 +14,7 @@ CPPFLAGS_x86_32 += -I$(src)/vendorcode/intel/edk2/UDK2017/MdePkg/Include/Ia32
 CPPFLAGS_x86_64 += -I$(src)/vendorcode/intel/edk2/UDK2017/MdePkg/Include/X64
 CPPFLAGS_common += -I$(src)/vendorcode/intel/edk2/UDK2017/MdePkg/Include
 CPPFLAGS_common += -I$(src)/vendorcode/intel/edk2/UDK2017/IntelFsp2Pkg/Include
+CPPFLAGS_common += -I$(src)/vendorcode/intel/edk2/UDK2017/MdeModulePkg/Include
```

Comment:
On Alder Lake I also had to add the VariableFormat.h file to `edk2-stable202005` bindings. See commit https://github.com/Dasharo/coreboot/commit/1f08d9bc633fc862c61dbc76859d879fe136b90c, should be able to cherry-pick it here.

Reply:
Looks like that will also be needed on Elkhart Lake.

Reply:
Cherry-picked [this commit](https://github.com/Dasharo/coreboot/commit/e70230817c08738a8ee0ca3d5d5af62986cf9933) instead, it's the same change, just had to find it in the tree.